_rules/SC1-2-3-video-alternative.md

```diff
@@ -0,0 +1,183 @@
+---
+name: video with audio has audio descriptions or transcript
+rule_type: composite
+description: |
+  This rule checks video elements with audio have an alternative for the video content as audio or as text.
+
+success_criterion:
+- 1.2.3 # Audio Description or Media Alternative (Prerecorded)
+
+atomic_rules:
+- SC1-2-video-audio-description
+- SC1-2-video-transcript
+- SC1-2-video-description-track
+- SC1-2-video-media-alternative
+
+authors:
+- Wilco Fiers
+- Brian Bors
+---
+
+## Test Procedure
+
+### Applicability
+
+The rule applies to every [non-streaming](#non-streaming) `video` element that is [visible](#visible), where the video contains audio.
+
+### Expectation
+
+For each test target, the outcome of at least one of the following rules is passed:
```

Comment:
After discussion with @WilcoFiers, this condition needs to change to something like:

> Each test target, needs to meet the expectations defined by at least one of the following rules:

This applies to *all* composite rules going forward, so not sure how to handle it here (leave for now until this is discussed with the group vs make the change now already for others to follow).

Reply:
Auto-WCAG rules are written according to the latest published draft of the ACT Rules Format. What you are referring to is a discussion that hasn't been resolved yet, let alone published. We can't expect rule writers to keep up with internal ACT discussions, even if those are on public mailing lists.

We'll update rules when there is a new public draft / CR.

Reply:
OK